site/app/templates/admin/Docker.twig

```diff
@@ -0,0 +1,39 @@
+<div class="content">
+	<div style="float:right">
+		<button class = "btn btn-default" onclick="updateDockerData()" id="refresh-docker-interface">Refresh Docker Information </button>
+		<p id = "docker-update-info"><em>(Last updated on : {{ last_updated | default("") }})</em></p>
+	</div>
+
+	<h1>Submitty Docker Interface</h1>
+
+	<h2>System Wide information:</h2>
+	{% if docker_info is not null %}
+		<p>Docker Version : {{ docker_info['ServerVersion'] }} </p>
+		<p>Architecture : {{ docker_info['Architecture'] }}</p>
+		<p>Total Memory : {{ docker_info['MemTotal']}}</p>
+		<p>Name : {{ docker_info['Name'] }}</p>
+		<p>OS : {{ docker_info['OperatingSystem'] }}</p>
+	{% endif %}
```

Comment:
Has someone expressed a desire to have this information? Unless there's a set need, we should not be showing this information.